packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -188,13 +192,75 @@ const useTrackStyles = makeStyles({
   }),
 });
 
+/**
+ * Styles for the mark slot
+ */
+const useMarksWrapperStyles = makeStyles({
+  marksWrapper: theme => ({
+    position: 'relative',
+    display: 'grid',
+    outline: 'none',
+    zIndex: '1',
+    whiteSpace: 'nowrap',
+    '& .ms-Slider-mark': {
+      // TODO: theme neutralStrokeOnBrand
+      background: 'white',
+    },
+
+    '& .first, .last': {
```

Comment:
The names `.first` and `.last` seem way too general, so there's risk that they could pick up extra styles if the Slider is used on a page where somebody happened to define those class names. From what I can tell, the preferred way to do this in converged would be to have separate styles like `firstMark` and `lastMark` which are conditionally applied (maybe @khmakoto can confirm).